tools/github-actions/download-build-output/readme.md

```diff
@@ -0,0 +1,7 @@
+# Download build output action
+
+## Overview
+Download and restore the `dist` folders from GitHub artifact (check ../upload-build-output)
```

Comment:
could be a proper link to `upload-build-output`'s readme

(and upload could also link the donwload's readme)